pkg/volume/scaleio/sio_volume.go

```diff
@@ -248,6 +248,10 @@ func (v *sioVolume) Provision() (*api.PersistentVolume, error) {
 	// setup volume attrributes
 	name := v.generateVolName()
 	capacity := v.options.PVC.Spec.Resources.Requests[api.ResourceName(api.ResourceStorage)]
+	if capacity.Value() == 0 {
```

Comment:
This patch chunk is removed in subsequent patch and IMO it should not be - PVC with requesting 0 bytes is wrong and should be probably rejected.